cms/templates/js/discussion-editor.underscore

```diff
@@ -0,0 +1,15 @@
+<form>
+<h3 class="modal-section-title">
+    <%- gettext('Discussion') %>
+</h3>
+<div class="modal-section-content discussion-enabled">
+    <label class="label">
+        <input type="checkbox" id="discussion_enabled" name="discussion_enabled" class="input input-checkbox" />
+        <% if (xblockInfo.isVertical()) { %>
+            <%- gettext('Enable discussion') %>
+        <% } else { %>
+            <%- gettext('Enable discussion for all child units') %>
+        <% } %>
```

Comment:
Are we still using this?

Reply:
Hmm, though I alerady removed this. Will fix.